Answers.md

```diff
@@ -0,0 +1,11 @@
+1.	If you saw this HTML: <div class="box box1 box2 box3"></div> which class has the most specificity weight?
+The class selector.box3 has the most specificity weight. 
```

Comment:
Yes, just to clarify. All of the elements technically have the same weight. Since they are all equal, the most weight is given to the last element which just so happen to be box3.